### PR TITLE
feat(de): launch German translation experiment

### PR DESCRIPTION
**Pull request by caugner**
## Summary

(MP-1641)

We launched the German locale as a [public preview on a separate test environment](https://github.com/orgs/mdn/discussions/741) in October 2024, and the next step is to expose the locale to more users by launching it as an experiment on production in November 2024, with the following constraints:

1. The locale should not be indexed by search engines.
2. The locale should be clearly marked as experimental and machine translated.

This PR achieves that, and includes the following changes:

1. The {stage,prod}-build workflows are updated to build `de` by checking out `mdn/translated-content-de` and moving the `files/de` folder into `mdn/translated-content`.
    - _Caveats_: This prevents `sync-translated-content` on `de`, and `tool gather-git-history` from fetching `de` history, but that's acceptable.
2. German translations are added in various places:
    - in the baseline indicator (i.e. the id of the BCD section),
    - the indicator for `en-US` links (untranslated pages),
    - the disclaimer at the top of every page about the locale being experimental and automatically translated,
    - the TOC header ("In this article"),
    - the article footer ("Help improve MDN")
    - macros (especially sidebars),
3. A short survey is added that is shown on all `de` pages.
    - _Related_: For this, the `src` field now supports a callback to create the survey URL dynamically in order to pass the current MDN page's URL as a survey parameter. 
4. Special treatment for German is added in two places:
    - in the `client`, the repo `translated-content` is manually replaced by `translated-content-de`,
    - during SSR, the robots value for German pages is set to `"noindex, follow"` to prevent search engines from indexing the pages.

---

## How did you test this change?

This branch is already being used to deploy to the test environment, and has been deployed [on stage since 2024-10-31](https://github.com/mdn/yari/actions/runs/11616102670).

### CODE REVIEW

File: kumascript/macros/GamesSidebar.ejs [hunk 1]
```diff
@@ -7,6 +7,61 @@ const webURL = `/${locale}/docs/Web/`;
 const appsURL = `/${locale}/docs/Web/Apps/`;
 
 const text = mdn.localStringMap({
+  "de": {
+    "Introduction": "Einführung",
+    "Anatomy": "Anatomie",
+    "APIs_for_game_development": "APIs für die Spieleentwicklung",
+    "Canvas": "Leinwand",
```
Comment (caugner) [inline kumascript/macros/GamesSidebar.ejs:14]:
```suggestion
    "Canvas": "Canvas",
```

File: kumascript/macros/GamesSidebar.ejs [hunk 2]
```diff
@@ -7,6 +7,61 @@ const webURL = `/${locale}/docs/Web/`;
...
+    "CSS": "CSS",
+    "Full_Screen": "Vollbild",
+    "Gamepad": "Gamepad",
+    "IndexedDB": "IndexedDB",
+    "JavaScript": "JavaScript",
+    "Pointer_Lock": "Mauszeiger Sperre",
```
Comment (caugner) [inline kumascript/macros/GamesSidebar.ejs:20]:
```suggestion
    "Pointer_Lock": "Pointer Lock",
```

File: kumascript/macros/JsSidebar.ejs [hunk 3]
```diff
@@ -10,6 +12,61 @@ function state(section) {
...
+    'Guide_Expressions': 'Ausdrücke und Operatoren',
+    'Guide_Numbers': 'Zahlen und Daten',
+    'Guide_Text': 'Textformatierung',
+    'Guide_RegExp': 'Reguläre Ausdrücke',
+    'Guide_Indexed_collections': 'Indexierte Sammlungen',
+    'Guide_keyed_collections': 'Gekennzeichnete Sammlungen',
+    'Guide_Objects': 'Arbeiten mit Objekten',
+    'Guide_Classes': 'Verwendung von Klassen',
+    'Guide_Promises': 'Verwendung von Promises',
+    'Guide_Typed_arrays': 'JavaScript-getypte Arrays',
```
Comment (caugner) [inline kumascript/macros/JsSidebar.ejs:33]:
```suggestion
    'Guide_Typed_arrays': 'Typisierte Arrays in JavaScript',
```

File: kumascript/macros/NonStandardBadge.ejs [hunk 4]
```diff
@@ -6,14 +6,16 @@
...
     "en-US": "Non-standard. Check cross-browser support before using.",
     "ko": "비표준. 사용하기전에 다른 브라우저에서도 사용 가능한지 확인 해주세요.",
     "zh-CN": "非标准。请在使用前检查跨浏览器支持。",
     "zh-TW": "非標準。請在使用前檢查跨瀏覽器支援。"
 });
 
 const abbreviation = mdn.localString({
-    "en-US": "Non-standard",
+    "de": "Nicht standardisiert",
+    "en-US": "Kein Standard",
```
Comment (caugner) [inline kumascript/macros/NonStandardBadge.ejs:18]:
```suggestion
    "en-US": "Non-standard",
```

File: kumascript/macros/NonStandardBadge.ejs [hunk 5]
```diff
@@ -6,14 +6,16 @@
 */
 
 const title = mdn.localString({
+    "de": "Kein Standard. Überprüfen Sie die Unterstützung in verschiedenen Browsern, bevor Sie es verwenden.",
```
Comment (caugner) [inline kumascript/macros/NonStandardBadge.ejs:9]:
```suggestion
    "de": "Nicht standardisiert. Überprüfen Sie die Unterstützung in verschiedenen Browsern, bevor Sie es verwenden.",
```

File: kumascript/macros/PreviousMenuNext.ejs [hunk 6]
```diff
@@ -15,6 +15,7 @@ const nextPage = $1?.replace(/ /g, "_");
 const mainMenu = $2?.replace(/ /g, "_");
 
 const previousNextStr = mdn.localString({
+    "de"   : [" Vorherige ", " Nächste  "],
```
Comment (caugner) [inline kumascript/macros/PreviousMenuNext.ejs:18]:
```suggestion
    "de"   : [" Zurück ", " Weiter  "],
```

File: kumascript/macros/ReadOnlyInline.ejs [hunk 7]
```diff
@@ -1,5 +1,6 @@
 <%
 var str = mdn.localString({
+    "de": "Nur lesen ",
```
Comment (caugner) [inline kumascript/macros/ReadOnlyInline.ejs:3]:
```suggestion
    "de": "Nur lesbar ",
```

File: kumascript/macros/SeeCompatTable.ejs [hunk 8]
```diff
@@ -1,6 +1,7 @@
 <%
 
 const str = mdn.localString({
+  "de": "<strong>Dies ist eine <a href='/de/docs/MDN/Contribute/Guidelines/Conventions_definitions#experimental'>experimentelle Technologie</a></strong><br />Überprüfen Sie die <a href='#browser_compatibility'>Browser-Kompatibilitätstabelle</a> sorgfältig, bevor Sie diese in der Produktion verwenden.",
```
Comment (caugner) [inline kumascript/macros/SeeCompatTable.ejs:4]:
```suggestion
  "de": "<strong>Dies ist eine <a href='/de/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete#experimentell'>experimentelle Technologie</a></strong><br />Überprüfen Sie die <a href='#browser_compatibility'>Browser-Kompatibilitätstabelle</a> sorgfältig, bevor Sie diese in der Produktion verwenden.",
```

File: kumascript/macros/WebAssemblySidebar.ejs [hunk 9]
```diff
@@ -3,6 +3,19 @@ var locale = env.locale;
 var baseURL = "/" + locale + "/docs/WebAssembly";
 
 var text = mdn.localStringMap({
+  'de': {
+    'WebAssembly_home_page' : 'WebAssembly-Startseite',
+    'Tutorials' : 'Anleitungen',
+    'WebAssembly_concepts' : 'WebAssembly-Konzepte',
+    'Compiling_to_wasm' : 'Kompilierung von C/C++ zu WebAssembly',
+    'Compiling_rust_to_wasm' : 'Kompilierung von Rust zu WebAssembly',
```
Comment (caugner) [inline kumascript/macros/WebAssemblySidebar.ejs:11]:
```suggestion
    'Compiling_to_wasm' : 'C/C++ zu WebAssembly kompilieren',
    'Compiling_rust_to_wasm' : 'Rust zu WebAssembly kompilieren',
```

File: kumascript/macros/WebAssemblySidebar.ejs [hunk 10]
```diff
@@ -3,6 +3,19 @@ var locale = env.locale;
...
+    'JavaScript_API' : 'Verwendung der WebAssembly JavaScript API',
```
Comment (caugner) [inline kumascript/macros/WebAssemblySidebar.ejs:12]:
```suggestion
    'JavaScript_API' : 'WebAssembly JavaScript API verwenden',
```

File: kumascript/macros/WebAssemblySidebar.ejs [hunk 11]
```diff
@@ -3,6 +3,19 @@ var locale = env.locale;
...
+    'Text_format' : 'Verständnis des WebAssembly-Textformats',
```
Comment (caugner) [inline kumascript/macros/WebAssemblySidebar.ejs:13]:
```suggestion
    'Text_format' : 'WebAssembly-Textformats verstehen',
```

File: kumascript/macros/WebAssemblySidebar.ejs [hunk 12]
```diff
@@ -3,6 +3,19 @@ var locale = env.locale;
...
+    'Text_format_to_wasm' : 'Umwandlung des WebAssembly-Textformats in wasm',
+    'Loading_and_running' : 'Laden und Ausführen von WebAssembly-Code',
```
Comment (caugner) [inline kumascript/macros/WebAssemblySidebar.ejs:15]:
```suggestion
    'Loading_and_running' : 'WebAssembly-Code laden und ausführen',
```

File: kumascript/macros/js_property_attributes.ejs [hunk 13]
```diff
@@ -14,6 +14,14 @@ const isEnumerable = $1 == 1;
 const isConfigurable = $2 == 1;
 
 const text = mdn.localStringMap({
+  "de": {
+    header: `Eigenschaftsattribute von <code>${env.title}</code>`,
```
Comment (caugner) [inline kumascript/macros/js_property_attributes.ejs:18]:
```suggestion
    header: `Eigenschaften der <code>${env.title}</code>-Property`,
```

File: kumascript/macros/secureContext_header.ejs [hunk 14]
```diff
@@ -14,6 +15,7 @@ var str_title = mdn.localString({
 });
 
 var str_desc = mdn.localString({
+  "de"   : "Diese Funktion ist nur in <a href='/" + lang + "/docs/Web/Security/Secure_Contexts'>sicheren Kontexten</a> (HTTPS) in einigen oder allen <a href='#browser_compatibility'>unterstützenden Browsern</a> verfügbar.",
```
Comment (caugner) [inline kumascript/macros/secureContext_header.ejs:18]:
```suggestion
  "de"   : "Diese Funktion ist nur in <a href='/" + lang + "/docs/Web/Security/Secure_Contexts'>sicheren Kontexten</a> (HTTPS) in einigen oder allen <a href='#browser-kompatibilität'>unterstützenden Browsern</a> verfügbar.",
```

File: libs/constants/index.js [hunk 15]
```diff
@@ -10,7 +10,7 @@ export const RETIRED_LOCALES = new Map(
     "bg",
     "bn",
     "ca",
-    "de",
+    //"de",
```
Comment (caugner) [inline libs/constants/index.js:13]:
```suggestion
```

File: server/translations.ts [hunk 16]
```diff
@@ -394,10 +394,7 @@ async function gatherL10NstatsSection({
   }
 
   function packageEdits(document) {
-    const commitURL = getLastCommitURL(
-      document.fileInfo.root,
-      document.metadata.hash
-    );
+    const commitURL = getLastCommitURL(locale, document.metadata.hash);
```
Comment (caugner) [inline server/translations.ts:397]:
```suggestion
    const commitURL = getLastCommitURL(document.locale, document.metadata.hash);
```

File: kumascript/macros/AccessibilitySidebar.ejs [hunk 17]
```diff
@@ -4,6 +4,16 @@ const locale = env.locale;
 const baseURL = `/${locale}/docs/`;
 
 const l10nStrings = mdn.localStringMap({
+  "de": {
+    "Accessibility" : "Barrierefreiheit",
+    "Guides" : "Anleitungen",
+    "Learn" : "Barrierefreiheit lernen",
+    "ARIA" : "ARIA",
+    "ARIA_guides" : "ARIA Anleitungen",
```
Comment (caugner) [inline kumascript/macros/AccessibilitySidebar.ejs:12]:
```suggestion
    "ARIA_guides" : "ARIA-Anleitungen",
```

File: kumascript/macros/AccessibilitySidebar.ejs [hunk 18]
```diff
@@ -4,6 +4,16 @@ const locale = env.locale;
...
+    "ARIA states and properties": "ARIA-Zustände und Eigenschaften",
```
Comment (caugner) [inline kumascript/macros/AccessibilitySidebar.ejs:13]:
```suggestion
    "ARIA states and properties": "ARIA-Zustände und -Eigenschaften",
```

File: kumascript/macros/AddonSidebarMain.ejs [hunk 19]
```diff
@@ -3,6 +3,15 @@ var locale = env.locale;
...
 
 var text = mdn.localStringMap({
+  'de': {
+    'WebExtensions': 'Browser-Erweiterungen',
+    'Themes': 'Themen',
+    'Community_and_Support': 'Community und Support',
+    'Channels': 'Kanäle',
+    'Blog': 'Add-ons Blog',
+    'Forums': 'Add-ons Foren',
+    '#Contact_us': 'Kontaktieren Sie uns'
```
Comment (caugner) [inline kumascript/macros/AddonSidebarMain.ejs:13]:
```suggestion
    '#Contact_us': 'Kontakt'
```

File: kumascript/macros/CSSRef.ejs [hunk 20]
```diff
@@ -1,6 +1,176 @@
 <%
 
 const text = mdn.localStringMap({
+  'de': {
+    'Tutorials': 'Tutorials',
+    'CSS_basics': 'CSS Grundlagen',
+    'CSS_first_steps': 'CSS erste Schritte',
+       'CSS_first_steps_overview': 'CSS erste Schritte Übersicht',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:8]:
```suggestion
       'CSS_first_steps_overview': 'CSS erste Schritte (Übersicht)',
```

File: kumascript/macros/CSSRef.ejs [hunk 21]
```diff
@@ -1,6 +1,176 @@
...
+       'What_is_CSS':  'Was ist CSS?',
+       'Getting_started_with_CSS': 'Erste Schritte mit CSS',
+       'How_CSS_is_structured': 'Wie CSS aufgebaut ist',
+       'How_CSS_works': 'Wie CSS funktioniert',
+       'Assessment_Styling_a_biography_page': 'Bewertung: Gestaltung einer Biografie-Seite',
+     'CSS_building_blocks': 'CSS Bausteine',
+       'CSS_building_blocks_overview': 'CSS Bausteine Übersicht',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:15]:
```suggestion
       'CSS_building_blocks_overview': 'CSS Bausteine (Übersicht)',
```

File: kumascript/macros/CSSRef.ejs [hunk 22]
```diff
@@ -1,6 +1,176 @@
...
+  'de': {
+    'Tutorials': 'Tutorials',
+    'CSS_basics': 'CSS Grundlagen',
+    'CSS_first_steps': 'CSS erste Schritte',
+       'CSS_first_steps_overview': 'CSS erste Schritte Übersicht',
+       'What_is_CSS':  'Was ist CSS?',
+       'Getting_started_with_CSS': 'Erste Schritte mit CSS',
+       'How_CSS_is_structured': 'Wie CSS aufgebaut ist',
+       'How_CSS_works': 'Wie CSS funktioniert',
+       'Assessment_Styling_a_biography_page': 'Bewertung: Gestaltung einer Biografie-Seite',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:13]:
```suggestion
       'Assessment_Styling_a_biography_page': 'Aufgabe: Gestaltung einer Biografie-Seite',
```

File: kumascript/macros/CSSRef.ejs [hunk 23]
```diff
@@ -1,6 +1,176 @@
...
+       'Attribute_selectors': 'Attribut-Selektoren',
+       'Pseudo-classes_and_pseudo-elements': 'Pseudo-Klassen und Pseudo-Elemente',
+       'Combinators': 'Kombinatoren',
+       'Cascade_and_inheritance': 'Kaskade und Vererbung',
+       'Cascade_layers': 'Kaskadenschichten',
+       'The_box_model': 'Das Box-Modell',
+       'Backgrounds_and_borders': 'Hintergründe und Rahmen',
+       'Handling_different_text_directions': 'Umgang mit verschiedenen Textrichtungen',
+       'Overflowing_content': 'Überlaufender Inhalt',
+       'CSS_values_and_units': 'CSS Werte und Einheiten',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:27]:
```suggestion
       'CSS_values_and_units': 'CSS-Werte und -Einheiten',
```

File: kumascript/macros/CSSRef.ejs [hunk 24]
```diff
@@ -1,6 +1,176 @@
...
+       'Sizing_items_in_CSS': 'Größenanpassung von Elementen in CSS',
+       'Images_media_and_form_elements': 'Bilder, Medien und Formularelemente',
+       'Styling_tables': 'Tabellen gestalten',
+       'Debugging_CSS': 'CSS debuggen',
+       'Organizing_your_CSS': 'CSS organisieren',
+       'Assessment_Fundamental_CSS_comprehension': 'Bewertung: Grundlegendes CSS-Verständnis',
+       'Assessment_Creating_fancy_letterheaded_paper': 'Bewertung: Erstellung von schickem Briefpapier',
+       'Assessment_A_cool_looking_box': 'Bewertung: Eine cool aussehende Box',
+     'Styling_text': 'Textgestaltung',
+       'Styling_text_overview': 'Textgestaltung Übersicht',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:37]:
```suggestion
       'Styling_text_overview': 'Textgestaltung (Übersicht)',
```

File: kumascript/macros/CSSRef.ejs [hunk 25]
```diff
@@ -1,6 +1,176 @@
...
+       'Fundamental_text_and_font_styling': 'Grundlegende Text- und Schriftgestaltung',
+       'Styling_lists': 'Listen gestalten',
+       'Styling_links': 'Links gestalten',
+       'Web_fonts': 'Web-Schriftarten',
+       'Assessment_Typesetting_a_community_school_homepage': 'Bewertung: Satz einer Community-Schulhomepage',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:42]:
```suggestion
       'Assessment_Typesetting_a_community_school_homepage': 'Aufgabe: Satz einer Community-Schulhomepage',
```

File: kumascript/macros/CSSRef.ejs [hunk 26]
```diff
@@ -1,6 +1,176 @@
...
+     'CSS_layout': 'CSS-Layout',
+       'CSS_layout_overview': 'CSS-Layout Übersicht',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:44]:
```suggestion
       'CSS_layout_overview': 'CSS-Layout (Übersicht)',
```

File: kumascript/macros/CSSRef.ejs [hunk 27]
```diff
@@ -1,6 +1,176 @@
...
+       'Flexbox': 'Flexbox',
+       'Grids' : 'Raster',
+       'Floats': 'Floats',
+       'Positioning': 'Positionierung',
+       'Multiple-column_layout': 'Mehrspalten-Layout',
+       'Responsive_design': 'Responsives Design',
+       'Beginners_guide_to_media_queries': 'Anfängerkurs zu Media-Queries',
+       'Legacy_layout_methods': 'Veraltete Layout-Methoden',
+       'Supporting_older_browsers': 'Unterstützung älterer Browser',
+       'Assessment_Fundamental_Layout_Comprehension' : 'Bewertung: Grundlegendes Layout-Verständnis',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:56]:
```suggestion
       'Assessment_Fundamental_Layout_Comprehension' : 'Aufgabe: Grundlegendes Layout-Verständnis',
```

File: kumascript/macros/CSSRef.ejs [hunk 28]
```diff
@@ -1,6 +1,176 @@
...
+      'Columns': 'Spalten',
+        'Basic_concepts_of_Multicol': 'Grundlagen von Multicol',
+        'Styling_columns': 'Spalten gestalten',
+        'Spanning_and_balancing': 'Spannen und Ausgleichen',
+        'Handling_overflow_in_Multicol': 'Überlauf in Multicol handhaben',
+        'Content_breaks_in_Multicol': 'Inhaltsumbrüche in Multicol',
+      'Conditional_rules': 'Bedingte Regeln',
+        'Using_feature_queries': 'Feature-Abfragen verwenden',
+      'Containment' : 'Eindämmung',
+        'Using_CSS_containment' : 'CSS-Eindämmung verwenden',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:85]:
```suggestion
      'Containment' : 'Containment',
        'Using_CSS_containment' : 'CSS-Containment verwenden',
```

File: kumascript/macros/Deprecated_Header.ejs [hunk 29]
```diff
@@ -20,6 +21,7 @@ const note = mdn.localString({
 });
 
 const desc = mdn.localString({
+  "de":    "Diese Funktion wird nicht mehr empfohlen. Obwohl einige Browser sie möglicherweise noch unterstützen, wurde sie möglicherweise bereits aus den relevanten Webstandards entfernt, befindet sich im Prozess der Entfernung oder wird nur aus Kompatibilitätsgründen beibehalten. Vermeiden Sie die Verwendung und aktualisieren Sie gegebenenfalls bestehenden Code; siehe die <a href='#browser-kompatibilität'>Kompatibilitätstabelle</a> am Ende dieser Seite, um Ihre Entscheidung zu treffen. Beachten Sie, dass diese Funktion jederzeit nicht mehr funktionieren kann.",
```
Comment (caugner) [inline kumascript/macros/Deprecated_Header.ejs:24]:
```suggestion
  "de":    "Diese Funktion wird nicht mehr empfohlen. Obwohl einige Browser sie möglicherweise noch unterstützen, kann sie bereits aus den relevanten Webstandards entfernt worden sein, befindet sich im Prozess der Entfernung oder wird nur aus Kompatibilitätsgründen beibehalten. Vermeiden Sie die Verwendung und aktualisieren Sie gegebenenfalls bestehenden Code; siehe die <a href='#browser-kompatibilität'>Kompatibilitätstabelle</a> am Ende dieser Seite, um Ihre Entscheidung zu treffen. Beachten Sie, dass diese Funktion jederzeit nicht mehr funktionieren kann.",
```

File: kumascript/macros/HTMLSidebar.ejs [hunk 30]
```diff
@@ -12,6 +12,50 @@ function state(section) {
...
+      'Introduction_to_HTML_overview': 'Übersicht zur Einführung in HTML',
+      'Getting_started_with_HTML': 'Erste Schritte mit HTML',
+      'Whats_in_the_head_metadata_in_HTML': 'Was ist im Kopf? Metadaten in HTML',
+      'HTML_text_fundamentals': 'Grundlagen der HTML-Texte',
+      'Creating_hyperlinks': 'Erstellen von Hyperlinks',
+      'Advanced_text_formatting': 'Erweiterte Textformatierung',
+      'Document_and_website_structure': 'Dokument- und Webseitenstruktur',
+      'Debugging_HTML': 'Debugging von HTML',
+      'Assessment_Marking_up_a_letter': 'Bewertung: Briefauszeichnung',
+      'Assessment_Structuring_a_page_of_content': 'Bewertung: Strukturierung einer Inhaltsseite',
```
Comment (caugner) [inline kumascript/macros/HTMLSidebar.ejs:28]:
```suggestion
      'Assessment_Marking_up_a_letter': 'Aufgabe: Briefauszeichnung',
      'Assessment_Structuring_a_page_of_content': 'Aufgabe: Strukturierung einer Inhaltsseite',
```

File: kumascript/macros/HTMLSidebar.ejs [hunk 31]
```diff
@@ -12,6 +12,50 @@ function state(section) {
...
+    'Multimedia_and_embedding': 'Multimedia und Einbettung',
+      'Multimedia_and_embedding_overview': 'Übersicht zu Multimedia und Einbettung',
+      'Images_in_HTML': 'Bilder in HTML',
+      'Video_and_audio_content': 'Video- und Audioinhalte',
+      'From_object_to_iframe_other_embedding_technologies': 'Von Object zu Iframe - andere Einbettungstechnologien',
+      'Adding_vector_graphics_to_the_web': 'Hinzufügen von Vektorgrafiken zum Web',
+      'Responsive_images': 'Responsive Bilder',
+      'Assessment_Mozilla_splash_page': 'Bewertung: Mozilla-Startseite',
```
Comment (caugner) [inline kumascript/macros/HTMLSidebar.ejs:36]:
```suggestion
      'Assessment_Mozilla_splash_page': 'Aufgabe: Mozilla-Startseite',
```

File: kumascript/macros/JsSidebar.ejs [hunk 32]
```diff
@@ -10,6 +10,61 @@ function state(section) {
 }
 
 var text = mdn.localStringMap({
+  'de': {
+    'Overview': 'JavaScript-Technologieübersicht',
```
Comment (caugner) [inline kumascript/macros/JsSidebar.ejs:14]:
```suggestion
    'Overview': 'JavaScript-Technologie (Übersicht)',
```

File: kumascript/macros/MDNSidebar.ejs [hunk 33]
```diff
@@ -1,6 +1,15 @@
 <%
 
 const l10nStrings = mdn.localStringMap({
+  "de": {
+    "history": "Geschichte",
+    "advisory_board": "Beirat",
```
Comment (caugner) [inline kumascript/macros/MDNSidebar.ejs:6]:
```suggestion
    "advisory_board": "Product Advisory Board",
```

File: kumascript/macros/SVGRef.ejs [hunk 34]
```diff
@@ -2,6 +2,14 @@
 const locale = env.locale;
 
 const text = mdn.localStringMap({
+  'de': {
+    'Tutorials': 'Tutorials',
+    'Reference': 'Referenz',
+    'Elements': 'Elemente',
+    'Attributes': 'Attribute',
+    'Guides': 'Leitfäden',
+    'Introducing SVG from scratch': 'SVG von Grund auf neu einführen'
```
Comment (caugner) [inline kumascript/macros/SVGRef.ejs:11]:
```suggestion
    'Introducing SVG from scratch': 'Einführung in SVG von Anfang an'
```

File: client/src/ui/organisms/article-actions/language-menu/index.tsx [hunk 35]
```diff
@@ -178,3 +180,17 @@ function LocaleRedirectSetting() {
     </form>
   );
 }
+
+function LocaleStatusIcon({ locale }: { locale: string }) {
+  switch (locale) {
+    case "de":
+      return (
+        <span title="This language is experimental.">
```
Comment (fiji-flo) [inline client/src/ui/organisms/article-actions/language-menu/index.tsx:188]: nit: This could be in German.

File: kumascript/macros/AvailableInWorkers.ejs
Comment (fiji-flo) [inline kumascript/macros/AvailableInWorkers.ejs:1]: Not sure when and how this changed, but double check this is in sync with: https://github.com/mdn/content/blob/rari/files/jsondata/L10n-Template.json#L210

File: kumascript/macros/WebExtAllExamples.ejs
Comment (fiji-flo) [inline kumascript/macros/WebExtAllExamples.ejs:1]: Please, make sure those are in https://github.com/mdn/content/blob/rari/files/jsondata/L10n-Template.json

File: kumascript/macros/WebExtExamples.ejs
Comment (fiji-flo) [inline kumascript/macros/WebExtExamples.ejs:1]: Please, make sure those are in https://github.com/mdn/content/blob/rari/files/jsondata/L10n-Template.json

**Review by fiji-flo — APPROVED** [review]
Looks good, please look at the comments before merging.

File: client/src/ui/organisms/article-actions/language-menu/index.tsx (hunk 35 above)
Comment (caugner) [reply client/src/ui/organisms/article-actions/language-menu/index.tsx:188]:
```suggestion
        <span title="Diese Sprache ist experimentell.">
```

File: client/src/ui/molecules/document-survey/surveys.ts [hunk 36]
```diff
@@ -67,4 +69,36 @@ export const SURVEYS: Survey[] = [
...
+    },
+    src: (doc: Doc) => {
+      const url = new URL(
+        "https://survey.alchemer.com/s3/8073795/Feedback-zur-deutschen-Version-von-MDN"
+      );
+      url.searchParams.set("referrer", doc.mdn_url);
+      return url.toString();
+    },
+    teaser:
+      "Wir arbeiten daran, die deutsche Übersetzung von MDN zu verbessern.",
```
Comment (caugner) [inline client/src/ui/molecules/document-survey/surveys.ts:97]:
```suggestion
      "Wir arbeiten noch an dieser experimentellen deutsche Übersetzung von MDN.",
```

**Pull request merged**